docs/settings/reporting-settings.asciidoc

```diff
@@ -261,7 +261,7 @@ For information about {kib} memory limits, see <<production, using {kib} in a pr
 [cols="2*<"]
 |===
 | `xpack.reporting.index`
-  | *deprecated* This setting is deprecated and will be removed in 8.0. Multitenancy by changing
```

Comment:
Same problem with the display of this one.

Reply:
I've solved this issue by sorting this list alphabetically, so the deprecated setting is no longer the first one.